### ISSUE TITLE
Pitch the project

### ISSUE THREAD

**Issue by unspentspace-admin** [unspentspace/melbourne#6]
Pitch the project to a group of people (e.g. the group who completed the survey) or pitch multiple times to individuals.

**Comment by M3rkl3-Tr33** [comment]
I might be able to secure a Venue/Room if we need ? 
I can call in a favour if we want it at a decent place. Any Dates in mind ? 
Could be a good oppourtunity

**Comment by Cyberpunk-Hardware** [comment]
> I might be able to secure a Venue/Room if we need ? 
I checked the one you discussed and it and looks good
> I can call in a favour if we want it at a decent place. Any Dates in mind ? 
I think we should do informal pitches before we hire the space? Can we get 10 github contributers first?
> Could be a good opportunity for Pledges and or commitments ??? 
Totally, so we need to formalise this first, i'm on it

**Comment by unspentspace-admin** [comment]
I think ptches for the project will work best with one-on-ones or very small groups. Initially, I had invisaged doing a video call with the 30 people who responded to the google form, but I think this will miss the opportunity to make quality connections.

For example, Nathan, who owns FastBitcoins and TwentyOne, lives in Kew. I know he'll be looking for a space soon. I also know he's not going to speak up in an online crowd, but would really appreciate a one on one.

So maybe we start to make appropriate groups/individuals to consult rather than lumping everyone in a presentation. Yes, it can be time effective doing one preso, but in the long run I'm not sure if you make as quality connections.

Thoughts @Cyberpunk-Hardware  @M3rkl3-Tr33 ?

**Comment by M3rkl3-Tr33** [comment]
Agreed, we can Target (Key) individuals and have a preso for the Masses if needed.
